packages/stark-ui/src/modules/dropdown/components/dropdown.component.ts

```diff
@@ -172,13 +222,60 @@ export class StarkDropdownComponent extends AbstractStarkUiComponent implements
 		if (changes["multiSelect"]) {
 			this.isMultiSelectEnabled = StarkComponentUtil.isInputEnabled(this.multiSelect);
 		}
+
+		if (changes["isDisabled"]) {
+			if (typeof this.dropdownFormControl !== "undefined") {
+				this.logger.warn(reactiveWarningMessage);
+			}
+
+			if (typeof this.formControl !== "undefined") {
+				if (this.isDisabled) {
+					this.formControl.disable();
+				} else {
+					this.formControl.enable();
+				}
+			}
+		}
+
+		if (changes["required"]) {
+			this.setDefaultBlank();
+			if (
+				!changes["required"].firstChange &&
+				typeof this.dropdownFormControl === "undefined" &&
```

Comment:
Same remark: maybe you can use the type-guard you use in the ngOnInit here instead